jupyterlab/staging/package.json

```diff
@@ -204,7 +204,7 @@
     "duplicate-package-checker-webpack-plugin": "^3.0.0",
     "fs-extra": "^10.1.0",
     "glob": "~7.1.6",
-    "handlebars": "^4.5.3",
+    "handlebars": "^4.7.8",
```

Comment:
Ah, just noticed we usually don't need to update the `jupyterlab/staging/package.json` file manually, as this is done during the release process.

So we should be able to undo this change and the one in `jupyterlab/staging/yarn.lock` too.